docs/data/toolpad/core/components/sign-in-page/ComponentsPropsSignIn.js

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { AppProvider, SignInPage } from '@toolpad/core';
+
+const providers = [{ id: 'credentials', name: 'Email and Password' }];
```

Comment:
Oh just one more thing, would be nice to show this provider in the preview for the credentials example too.